.github/workflows/jobs.yml

```
@@ -0,0 +1,31 @@
+on:
+    push
+
+jobs:
+    lint:
+        name: Lint Codebase
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout Repository
+              uses: actions/checkout@v4
+            - name: Install Dependencies
+              uses: ./.github/actions/install
```

actionlint (1.7.8)

description is required in metadata of "Install Action" action at "/home/jailuser/git/.github/actions/install/action.yml"

(action)